spaceR/SpacerTests/SpaceRTests.swift

```diff
@@ -1,6 +1,6 @@
 //
-//  spaceRTests.swift
-//  spaceRTests
+//  SpaceRTests.swift
```

Comment:
Pretty sure I have seen these changes before, please make sure that you pull develop before you start a new branch 😄

Reply:
I had to start a new branch off the old one and not develop because the old one's PR took like 7 days to be approved and I didn't want to keep coding in the old one..What should i have done in that situation in case it happens again?